src/cheri-pte-ext.adoc

```diff
@@ -26,7 +26,9 @@ NOTE: The description below assumes that {cheri_pte_ext_name} has been implement
 
 The minimum level of PTE support is to set CW to 1 in all PTEs intended for storing capabilities (i.e. private anonymous mappings) and leave <<sstatusreg_pte,sstatus>>.UCRG and CRG in all PTEs set to 0, which will allow capabilities with their tags set to be loaded and stored successfully.
 
-NOTE: Hardware initiated memory accesses from the page-table walker are not checked by a capability.
+If <<section_cheri_disable,CRE>>=0 then the PTE.CW, PTE.CRG bits become _reserved_ when checking PTE permissions on memory accesses.
+
+NOTE: Hardware initiated memory accesses from the page-table walker are not checked by a capability, and also do not consider PTE.CW, PTE.CRG bits to be _reserved_.
```

Comment:
```suggestion
NOTE: Hardware initiated memory accesses from the page-table walker are not checked by a capability.
```

Restoring old wording, this behaviour is no different from normal PTE reserved bit checking.